src/components/app.js

```diff
@@ -10,6 +10,13 @@ import Footer from './footer';
 let store = createStore();
```

Comment:
`location` is available during prerendering ([source](https://github.com/preactjs/preact-cli/blob/master/packages/cli/lib/lib/webpack/prerender.js#L14)):

```suggestion
let store = createStore({
  location: location.pathname
});
```

Reply:
Sweeet!! 💯